pkgs/misc/vim-plugins/generated.nix

```diff
@@ -5332,12 +5332,12 @@ let
 
   vim-clap = buildVimPluginFrom2Nix {
     pname = "vim-clap";
-    version = "2021-04-17";
+    version = "2021-04-25";
     src = fetchFromGitHub {
       owner = "liuchengxu";
       repo = "vim-clap";
-      rev = "8e13b23d69549c95d9c223ea5c2487d5dd9558f7";
-      sha256 = "1biiq07dhrz9vhk0yg3zkkv3329nyla6lp8kavdzqrvqg0hsbr2j";
+      rev = "1afdd263a862bae0641f565e3f2952e1c01cec43";
+      sha256 = "0c2jrz02dsdykc3xxqw1yfnllmrpwzs6ygjqcclghw5mygfc3xcg";
```

Comment:
looks like you need to update the `cargoSha256` in overrrides

```
builder for '/nix/store/jnav35qdfblsk868q0m3q1gdgvv1sg78-maple.drv' failed with exit code 1; last 10 log lines:

  ERROR: cargoSha256 is out of date

  Cargo.lock is not the same in maple-vendor.tar.gz

  To fix the issue:
  1. Use "0000000000000000000000000000000000000000000000000000" as the cargoSha256 value
  2. Build the derivation and wait for it to fail with a hash mismatch
  3. Copy the 'got: sha256:' value back into the cargoSha256 field
cannot build derivation '/nix/store/aybp3inz3gg64955pk3fb2jyv0qacyg6-vimplugin-vim-clap-2021-04-25.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/k2zykjrd15fb8pgxzp91jarzj0yrscki-env.drv': 1 dependencies couldn't be built
error: --- Error ------------------------------------------------------------------------------------------------------- nix
build of '/nix/store/k2zykjrd15fb8pgxzp91jarzj0yrscki-env.drv' failed
```

Reply:
Ouch :/ Fixed